src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -212,7 +220,8 @@ const Posts = () => {
 
 Posts.propTypes = {
   className: PropTypes.string,
-  posts: PropTypes.array,
+  searchResults: PropTypes.array,
+  searchPostResults: PropTypes.array,
```

Comment:
Why do we have two props here?  You only appear to use `searchResults` in the component.  Let's remove `searchPostResults`

Reply:
Another question re: naming - if what we're passing into this component on props are actually `posts` then maybe that's the right name vs. `searchResults` (i.e.,. it's more accurate, generic, and reusable for the `Posts` component)?

```jsx
<!-- No posts are being passed, load them internally -->
<Posts />

<!-- We are passing posts to be rendered -->
<Posts posts={...} />
```

Reply:
I wanted to use `posts`, however we are using that variable already in [posts](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/src/components/Posts/Posts.jsx#L46) to declare an empty array and append to as we get more posts. React also yells at me to use object destructuring, unless you want me to ignore it and use something like `const (variable) = props.posts`

Reply:
Gotcha.  Let's call it `searchResults` then, and you can use destructing if you want, sure.  Remove the second prop in proptypes.